scripts/makeheader

```diff
@@ -3,6 +3,9 @@ from __future__ import print_function
 import re
 import os
 
+# expected format:
+# #define XF86XK_FooBar _EVDEVK(0x123) /* some optional comment */
+evdev_pattern = re.compile(r'^#define\s+XF86XK_(?P<name>\w+)\s+_EVDEVK\((?P<value>0x[0-9A-F]+)\)')
```

Comment:
Just to be on the safe side, would allow lowercase in the hex:

```suggestion
evdev_pattern = re.compile(r'^#define\s+XF86XK_(?P<name>\w+)\s+_EVDEVK\((?P<value>0x[0-9A-Fa-f]+)\)')
```

Reply:
updated in latest version (both instances, there was another regex further down), thanks.